install/kubernetes/cilium/templates/cilium-operator/clusterrole.yaml

```diff
@@ -105,8 +142,27 @@ rules:
   - create
   - get
   - list
-  - update
   - watch
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - update
+  resourceNames:
+  - ciliumbgploadbalancerippools.cilium.io
+  - ciliumbgppeeringpolicies.cilium.io
+  - ciliumclusterwideenvoyconfig.cilium.io
```

Comment:
unlike cilium-agent, cilium operator is doing nothing with ciliumclusterwideenvoyconfig resource, we can drop it here.

Reply:
I assume you meant that we could drop it from here?
```patch
 - apiGroups:
   - cilium.io
   resources:
   - ciliumendpointslices
   - ciliumenvoyconfigs
-  - ciliumclusterwideenvoyconfigs
   verbs:
   - create
   - update
   - get
   - list
   - watch
   - delete
```

just a small note that this line where you left the review comment is specifically for the update of the CRD resource, which Cilium Operator creates and should be able to update the CRD itself.

Reply:
you are right :100: , I was pointing to the wrong line.